src/julia-config-p.h.in

```diff
@@ -0,0 +1,9 @@
+//
+
+#ifndef JULIA_CONFIG_P_H
+#define JULIA_CONFIG_P_H
+
+#define JL_SYSTEM_IMAGE_PATH "@JL_SYSTEM_IMAGE_PATH@"
```

Comment:
This should be the same with before? (with stringreplace as post install script?)